pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -44,9 +44,9 @@ qtModule {
   # which cannot be set at the same time as -Wformat-security
   hardeningDisable = [ "format" ];
 
-  postPatch = ''
+  postPatch =
+    lib.optionalString (lib.versionAtLeast qtCompatVersion "5.15") ''
     # Patch Chromium build tools
-    (
```

Comment:
If we keep those I think we can get around the mass rebuild. Also this should maybe go to staging-next.